hugegraph-loader/assembly/static/bin/hugegraph-spark-loader.sh

```diff
@@ -25,7 +25,7 @@ get_params "$@"
 echo "engine_params: $ENGINE_PARAMS"
 echo "hugegraph_params: $HUGEGRAPH_PARAMS"
 
-ASSEMBLY_JAR_NAME=$(find "${LIB_DIR}" -name 'hugegraph-loader*.jar')
+ASSEMBLY_JAR_NAME=$(find "${LIB_DIR}" -name 'apache-hugegraph-loader-incubating*.jar')
```

Comment:
```suggestion
ASSEMBLY_JAR_NAME=$(find "${LIB_DIR}" -name '*hugegraph-loader-incubating*.jar')
```

try this way if test **OK**, just a suggestion, not required (to avoid filename changed)